tokio/src/net/driver/reactor/dispatch/tid.rs

```diff
@@ -1,168 +0,0 @@
-use super::{page, Pack};
```

Comment:
Thread ID is not currently needed for the single shard slab. At some point, multi-shards will be added back, but I expect the thread tracking strategy to be significantly different.